test/sql/create_hypertable.sql

```diff
@@ -441,4 +441,63 @@ ALTER TABLE sample_table DROP COLUMN name;
 SELECT * FROM create_hypertable('sample_table', 'time');
 
 -- cleanup
-DROP TABLE sample_table CASCADE;
\ No newline at end of file
+DROP TABLE sample_table CASCADE;
+
+-- github issue 4684
+CREATE TABLE regular(
+   id INT NOT NULL,
+   dev INT NOT NULL,
+   value INT,
+   CONSTRAINT cstr_regular_pky PRIMARY KEY (id)
+) PARTITION BY HASH (id);
```

Comment:
The title of the PR seems to indicate that it problem is specific to hash-partitioned tables. Is it really the case that  this problem occurs only with hash-partitioned tables or does it also happen with LIST and RANGE partitioned ones as well? 

In either case, it might be good to test list and range partitioning as well. If this is a general problem with partitioned tables, then the title of the PR might need to be updated.

Reply:
Thanks. Issue exists for all types of partitioned tables. Tested for LIST, RANGE and HASH based partitioned tables and added the same in create_hypertable.sql.
Also removed reference to hash partition in commit message.